docs/sql-migration-guide.md

```diff
@@ -59,7 +59,7 @@ license: |
  
   - In Spark 3.0, you can use `ADD FILE` to add file directories as well. Earlier you could add only single files using this command. To restore the behavior of earlier versions, set `spark.sql.legacy.addSingleFileInAddFile` to `true`.
 
-  - In Spark 3.0, `SHOW TBLPROPERTIES` throws `AnalysisException` if the table does not exist. In Spark version 2.4 and below, this scenario caused `NoSuchTableException`. Also, `SHOW TBLPROPERTIES` on a temporary view causes `AnalysisException`. In Spark version 2.4 and below, it returned an empty result.
+  - In Spark 3.0, `SHOW TBLPROPERTIES` throws `AnalysisException` if the table does not exist. In Spark version 2.4 and below, this scenario caused `NoSuchTableException`.
```

Comment:
maybe we can remove it. Exception class change doesn't worth a migration guide item.

Reply:
This exception class change was brought up in this comment: https://github.com/apache/spark/pull/26921#discussion_r368060187. Can I still remove it?

Reply:
I don't think error message/exception type change is a breaking change. @dongjoon-hyun what do you think?